pkg/sql/plan/opt_misc.go

```diff
@@ -1048,6 +1048,15 @@ func (builder *QueryBuilder) pushTopDownToLeftJoin(nodeID int32) {
 	}
 
 	nodePushDown = DeepCopyNode(node)
+
+	if nodePushDown.Offset != nil {
+		offset := nodePushDown.Offset.GetLit().GetI64Val()
```

Comment:
You should handle parameters in PREPARE stmt here

Reply:
fixed